lib/wordmove/movefile.rb

```diff
@@ -26,6 +26,17 @@ def fetch(verbose = true)
       YAML.safe_load(ERB.new(File.read(found)).result, [], [], true).deep_symbolize_keys!
     end
 
+    def dotenv(cli_options = {})
+      env = cli_options['environment'] || cli_options[:environment]
```

Comment:
That makes sense. I believe my goal with that was to avoid unnecessary function calls. It does seem to be a bit of a footgun for stability, though. I'll go ahead and use your version.